spec/factories/omniauth.rb

```diff
@@ -0,0 +1,37 @@
+FactoryGirl.define do
+
+  factory :auth_hash, class: OmniAuth::AuthHash do
+    skip_create
```

Comment:
FYI, you can also keep factory_girl from saving the object by using `FactoryGirl.build` rather than `FactoryGirl.create` to generate it. `skip_create` is a nice safeguard though.